dapps/digital-twin-detail/src/components/container/List.vue

```diff
@@ -59,6 +60,7 @@
             >{{ '_twin-detail.data.list.show-all' | translate }}</span>
           </b-dropdown-item>
           <b-dropdown-item
+            v-if="isEditable"
```

Comment:
I think we agreed on greying it out yesterday? We can disable it just like `Show All`.